### PR TITLE
database loading and tests

### PR DESCRIPTION
**Pull request by char26**

### CODE REVIEW

File: python/gink/__main__.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 from importlib import import_module
 from os import environ
 from json import dumps
+import datetime # allows setting of datetime objects from CLI
+# TODO: don't import all of datetime
```
Comment (char26) [inline python/gink/__main__.py:13]: A container contents dump with datetime objects will look something like datetime.datetime(3, 12, 14, ...), so using `from datetime import datetime` won't work for loading. This is something to figure out, but maybe can be figured out in a separate PR.

File: python/gink/impl/lmdb_store.py [hunk 2]
```diff
@@ -314,11 +315,6 @@ def get_reset_changes(self, to_time: MuTimestamp, container: Optional[Muid],
                     for change in self._container_reset_changes(to_time, muid, seen, txn):
                         yield change
                     cursor_placed = containers_cursor.next()
-                # then loop over the "magic" pre-defined
-                for behavior in [DIRECTORY, SEQUENCE, BOX, KEY_SET]:
-                    muid = Muid(-1, -1, behavior)
-                    for change in self._container_reset_changes(to_time, muid, seen, txn):
-                        yield change
```
Comment (char26) [inline python/gink/impl/lmdb_store.py:321]: The global containers are already looped over in the first container_reset_changes call, so this was causing duplicate resets.

I'm not sure what I did for this to just now start failing the test, but restoring a previously reset global sequence was appending the same value twice.

**Review by darinmcgill — APPROVED** [review]
Looks good, thanks!
